backend/src/v5/middleware/sso/index.js

```diff
@@ -31,10 +31,13 @@ Sso.redirectWithError = (res, url, errorCode) => {
 	res.redirect(urlRedirect.href);
 };
 
-Sso.setSessionReferer = async (req, res, next) => {
-	if (req.headers.referer) {
-		req.session.referer = getURLDomain(req.headers.referer);
-	}
+Sso.setSsoInfo = async (req, res, next) => {
```

Comment:
probably better to call this set sessionInfo as the context we're in is SSO (you could argue every function here is setting sso Info 😆 )